sdk/eventhub/event-processor-host/package.json

```diff
@@ -46,7 +46,7 @@
     "mocha": "^5.2.0",
     "mocha-junit-reporter": "^1.18.0",
     "mocha-multi": "^1.0.1",
-    "nyc": "^13.3.0",
+    "nyc": "^14.0.0-rc.1",
```

Comment:
Reminder:
Once this is merged, please create an issue to update `^14.0.0-rc.1` to `^14.0.0`

Reply:
The issue should refer to all of the packages we changed.

Reply:
NYC 14.0.0 was released last night so I am updating this PR, and I don't think we need a follow-up issue unless you disagree @AlexGhiondea.  I'm not sure what you mean by "issue should refer to all of the packages we changed".